checkQC/app.py

```diff
@@ -1,22 +1,22 @@
 
-import yaml
 import sys
 
 from checkQC.qc_engine import QCEngine
+from checkQC.config import get_config
+from checkQC.run_type_recognizer import RunTypeRecognizer
 
 
 def start():
 
-    with open("config/config.yaml") as stream:
-        config = yaml.load(stream)
+    config = get_config("config/config.yaml")
 
-    machine_type = "hiseq2500_rapid"
-    read_length = '100-120'
-    run_mode = "paired_end"
+    runfolder = "./tests/resources/MiSeqDemo"
 
-    handler_config = config[machine_type][read_length][run_mode]["handlers"]
+    run_type_recognizer = RunTypeRecognizer(config=config, runfolder=runfolder)
+    instrument_type = run_type_recognizer.instrument_type()
+    read_length = run_type_recognizer.read_length()
```

Comment:
Ah sorry, discovered a new thing now. 
As I understand it, read_length() will (if paired-end data) return `<readlenght1>-<readlength2>`. This will not work when declaring the handler_config (row 19), right? The read length interval has to be determined before extracting the handler config.